pkg/cmd/install.go

```diff
@@ -101,15 +106,28 @@ func MakeInstall() *cobra.Command {
 		fmt.Printf("Result: %s %s\n", string(res.StdOut), string(res.StdErr))
 
 		absPath, _ := filepath.Abs(localKubeconfig)
-		fmt.Printf("Saving file to: %s\n", absPath)
 
-		kubeconfig := strings.Replace(string(res.StdOut), "localhost", ip.String(), -1)
+		kubeconfig := []byte(strings.Replace(string(res.StdOut), "localhost", ip.String(), -1))
 
-		writeErr := ioutil.WriteFile(absPath, []byte(kubeconfig), 0600)
-		if writeErr != nil {
+		if merge {
+			// Create a merged kubeconfig
+			kubeconfig, err = mergeConfigs(absPath, []byte(kubeconfig))
+			if err != nil {
+				return err
+			}
+		}
+		// Create a new kubeconfig
+		if writeErr := writeConfig(absPath, []byte(kubeconfig), false); writeErr != nil {
 			return writeErr
 		}
 
+		// Switch context
+		fmt.Println("Switching to the current context: default")
+		cmd := exec.Command("kubectl", "config", "set", "current-context", "default")
```

Comment:
Further.. how do you know that the context is going to be called `default`? What if there was already a file with a context called `default`?

Perhaps we should omit this code for the first release?

Reply:
You're right. Didn't think about this